frontend/src/components/task-card-modal/Markdown.sass

```diff
@@ -0,0 +1,13 @@
+*
```

Comment:
Staraj się tak nie robić. Cały sass kompiluje się do jednego pliku CSS, więc te style będą dostępne w całej aplikacji i zmieniłeś style wszystkich tagów `tr`, `table` itp w całym projekcie

Reply:
Powinno być tutaj ograniczenie do jakiejś klasy pod którą będą się znajdować te elementy np.:

```sass
.ModalComment-content
  *
    font-family: "Roboto","Helvetica","Arial",sans-serif
  table
    border-collapse: collapse
    tr:nth-child(2n)
      background: #f6f8fa
```

Reply:
To, że importujesz konkretny plik sass w konkretnym pliku JS, nie znaczy, że style będą zastosowane tylko w tym komponencie. To mówi kompilatorowi `esbuild`, że ma dołączyć ten plik `sass` do wynikowego arkusza `.css`. Dzielimy style na osobne pliki `.sass` dla każdego komponentu osobno, żeby było wygodnie edytować style, które dotyczą danego komponentu. To był powód dla którego mówiłem, żeby każde style były pod główną klasą komponentu -- żeby style nie wylewały się na resztę aplikacji

Reply:
A jak chcesz zmienić jakieś style, które dotyczą całej aplikacji, a nie jednego komponentu to możesz je dodać do `frontend/src/App.sass`

Reply:
albo przynajmniej dać je do osobnego pliku `.sass` i importować je we `frontend/src/App.js`

Reply:
Poprawiłem i zmerge'owałem